common/src/main/java/com/wynntils/overlays/minimap/MinimapOverlay.java

```diff
@@ -76,13 +76,7 @@ public class MinimapOverlay extends Overlay {
     public final Config<CompassRenderType> showCompass = new Config<>(CompassRenderType.ALL);
 
     @Persisted
-    public final Config<Boolean> renderRemoteFriendPlayers = new Config<>(true);
-
-    @Persisted
-    public final Config<Boolean> renderRemotePartyPlayers = new Config<>(true);
-
-    @Persisted
-    public final Config<Float> remotePlayersHeadScale = new Config<>(0.4f);
+    public final Config<Float> remotePlayersHeadScale = new Config<>(0.8f);
```

Comment:
This got me thinking. It feels wrong to treat player map features specially like this. Is this really a useful thing to change? If so, maybe all map features should have a iconScale attribute?

Reply:
That's something we should decide on right now. I've just started working on natively using map features (https://github.com/Wynntils/Artemis/pull/2578), and I've had to remove "poiScale" configs, because we can't treat them. Icons can be scaled for map locations, but what about map paths, and map areas? Map areas can have icons, in the center, but map paths, thats a weirder story. Overall, I have many ideas for map paths and map areas now, so you'll probably see an endless stream of PRs from me, depending on each other.

I think if we want to keep "icon scale", we should definitely make it an attribute.

Reply:
We could keep a global icon scale, as a config of the main map and minimap, that will scale all icons. That would be good to have, I think.

I am not completely sure it makes sense to scale other icons to different sizes. Was it ever a good idea to make player heads differently sized from other icons? If we think it was, and want to keep that, then we will need an iconScale attribute.